devices.js

```diff
@@ -4639,9 +4639,11 @@ const devices = [
             await firstEndpoint.configureReporting('msPressureMeasurement', pressureBindPayload);
         },
         exposes: [
-            e.co2(), e.temperature(), e.humidity(), e.pressure(), exposes.binary('led_feedback', 'rw', 'ON', 'OFF'),
-            exposes.binary('enable_abc', 'rw', 'ON', 'OFF'), exposes.numeric('threshold1', 'rw').withUnit('ppm'),
-            exposes.numeric('threshold2', 'rw').withUnit('ppm'),
+            e.co2(), e.temperature(), e.humidity(), e.pressure(),
+            exposes.binary('led_feedback', exposes.access.ALL, 'ON', 'OFF'),
+            exposes.binary('enable_abc', exposes.access.ALL, 'ON', 'OFF'),
```

Comment:
Can you also add description for this fields?

Reply:
Description like this it ok?
All LEDs feedback
ABC (Automatic Baseline Correction) algorithm
CO2 level when exceeded, LED2 blinks
CO2 when exceeded, LED2 ON, LED3 blinks

Reply:
Enable LEDs feedback 
Enable ABC (Automatic Baseline Correction)

Reply:
done

Reply:
Led thresholds descriptions still quite odd, but I can't figure out better wording

Reply:
Maybe, warning CO2 level and danger CO2 level ?

Reply:
Sounds more reasonable

Reply:
Warning (LED2) co2 level
Critical (LED3) co2 level

Reply:
btw were are this attributes? They needed to offset fake bme280 sensors measurements (mine are showing 33C instead 29C, to offset will be -4)
temperature_offset
pressure_offset
humidity_offset

Reply:
my fault. lost them at all. 
already added

Reply:
Can this be merged?

Reply:
This update devices.js only draws a nice view on the exposes, tab but doesn't work.
I will update fromZigbee.js and toZigbee.js in a few days, and upload to this PR.
I think it's worth merge after that.

Reply:
This update devices.js only draws a nice view on the exposes, tab but doesn't work.
I will update fromZigbee.js and toZigbee.js in a few days, and upload to this PR.
I think it's worth merge after that.